CHANGELOG.md

```diff
@@ -1,24 +1,32 @@
-Version 0.4.0
+Version 0.19-UNRELEASED
 
-* Evan Sampson added Holdridge life zones model
+* Added export support using libgdal
 
-Version 0.4.1
+Version 0.18
 
-* Generation of grayscale heightmap
+* Merged Lands and WorldSynth into one WorldEngine
+* River and its erosion are now taken into account
+* Total restructuring of code including many bug fixes.
+* Windows/OSX/Linux binary builds
 
-Version 0.4.2
+Version 0.5.4
```

Comment:
I meant: we should remove these references to versions < 0.18.0 because they are not related to WorldEngine